win32/tests/threadpool_win32_ut/CMakeLists.txt

```diff
@@ -19,9 +19,12 @@ set(${theseTestsName}_h_files
 ../../../interfaces/inc/azure_c_pal/string_utils.h
 )
 
-build_c_tests(${theseTestsName} ON "tests/azure_c_pal/win32" ADDITIONAL_LIBS pal_interfaces synchronization)
+build_test_artifacts(${theseTestsName} ON "tests/azure_c_pal/win32" ADDITIONAL_LIBS pal_interfaces synchronization)
 
-set_target_properties(${theseTestsName}_exe PROPERTIES LINK_FLAGS "/ignore:4217")
-if(use_cppunittest)
-    set_target_properties(${theseTestsName}_dll PROPERTIES LINK_FLAGS "/ignore:4217")
+if("${building}" STREQUAL "exe")
+    set_target_properties(${theseTestsName}_exe PROPERTIES LINK_FLAGS "/ignore:4217")
+endif()
+
+if("${building}" STREQUAL "dll")
+        set_target_properties(${theseTestsName}_dll PROPERTIES LINK_FLAGS "/ignore:4217")
```

Comment:
>     [](start = 4, length = 4)

nitpick: extra indent

Reply:
Here and in the other ut projects

---
In reply to: [462446064](https://github.com/Azure/azure-c-pal/pull/21#discussion_r462446064) [](ancestors = 462446064)

Reply:
eaten some spaces.

---
In reply to: [462446268](https://github.com/Azure/azure-c-pal/pull/21#discussion_r462446268) [](ancestors = 462446268,462446064)